config/application.yml

```diff
@@ -0,0 +1,2 @@
+SENDGRID_USERNAME: app64683711@heroku.com
```

Comment:
Per my Bloc message, these keys need to be removed, Sendgrid keys deactivated, new ones created and application.yml added to .gitignore.